pom.xml

```
@@ -146,6 +147,26 @@
 				<artifactId>apollo-openapi</artifactId>
 				<version>${apollo-java.version}</version>
 			</dependency>
+			<dependency>
+				<groupId>com.ctrip.framework.apollo.apollo-audit</groupId>
+				<artifactId>apollo-audit-annotation</artifactId>
+				<version>${revision}</version>
+			</dependency>
+			<dependency>
+				<groupId>com.ctrip.framework.apollo.apollo-audit</groupId>
+				<artifactId>apollo-audit-impl</artifactId>
+				<version>${revision}</version>
+			</dependency>
+			<dependency>
+				<groupId>com.ctrip.framework.apollo.apollo-audit</groupId>
+				<artifactId>apollo-audit-api</artifactId>
+				<version>${revision}</version>
+			</dependency>
+			<dependency>
+				<groupId>com.ctrip.framework.apollo.apollo-audit</groupId>
+				<artifactId>apollo-audit-spring-boot-starter</artifactId>
+				<version>${revision}</version>
```

```suggestion
				<version>${project.version}</version>
```
`revision` here will cause some problem to maven, use `project.version` instead of it.